src/GraphQL.sln

```diff
@@ -84,6 +84,10 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "GraphQL.SystemTextJson", "G
 EndProject
 Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Serialization", "Serialization", "{7BA2B19D-2839-4C7D-9BB0-4F3ADE691ADD}"
 EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "GraphQL.Extensions.DI.Microsoft", "GraphQL.Extensions.DI.Microsoft\GraphQL.Extensions.DI.Microsoft.csproj", "{806E7A14-668C-46A9-9C68-E98A49400E08}"
```

Comment:
I would shorten this name to `GraphQL.MicrosoftDI` as it is done for other projects.

Reply:
The same suggestion for `GraphQL.Extensions.DI.Microsoft` namespace.